pkg/asset/cluster/tfvars.go

```diff
@@ -181,6 +181,12 @@ func (t *TerraformVariables) Generate(parents asset.Parents) error {
 		}
 	}
 
+	lengthBootstrapFile := int64(len(bootstrapIgn))
+	if installConfig.Config.Platform.Azure != nil && installConfig.Config.Platform.Azure.CustomerManagedKey != nil &&
+		installConfig.Config.Platform.Azure.CustomerManagedKey.UserAssignedIdentityKey != "" {
+		lengthBootstrapFile = (((lengthBootstrapFile / 512) + 1) * 512)
```

Comment:
If the size is 512, you're returning here 1024. Shouldn't 512 be fine (no padding needed)?

Reply:
Fixed this.